src/main/scala/centaur/test/formulas/TestFormulas.scala

```diff
@@ -56,7 +56,7 @@ object TestFormulas {
   def runSequentialCachingWorkflows(firstAttempt: Workflow, secondAttempt: Workflow) = {
     for {
       firstWF <- runWorkflowUntilTerminalStatus(firstAttempt, Succeeded)
-      _ <- runSuccessfulWorkflowAndVerifyCacheMetadata(secondAttempt, firstWF.id)
+      _ <- runSuccessfulWorkflowAndVerifyCacheMetadata(secondAttempt, firstWF.id.id)
```

Comment:
That's the unfortunate side effect of my love of typing everything. Instead of having an I'd slid type uuid t is a custom type workflowid which wraps a uuid, but if you need that uuid you need to dig in and grab that value as well